scripts/create-docker-container.bash

```diff
@@ -40,6 +43,8 @@ docker pull "${IMAGE}"
 docker rm -f $CONTAINER_NAME
 docker run -it \
   --env "DB=${DB}" \
+  --env "DB_USER=${DB_USER}" \
+  --env "DB_PASSWORD=${DB_PASSWORD}" \
```

Comment:
@geofffranks This should also be the default behavior. It will [extract default params](https://github.com/cloudfoundry/diego-release/blob/develop/scripts/docker/test.bash#L13) by using the [diego release's default](https://github.com/cloudfoundry/wg-app-platform-runtime-ci/blob/main/diego-release/default-params/run-bin-test/linux.yml). Were you seeing something different?